mocks/terms.ts

```diff
@@ -0,0 +1,14 @@
+export default [
```

Comment:
This is just used for testing - we can add other mocks in this directory later on.